src/compiler/transformers/taggedTemplate.ts

```diff
@@ -81,6 +81,8 @@ namespace ts {
         // Examples: `\n` is converted to "\\n", a template string with a newline to "\n".
         let text = node.rawText;
         if (text === undefined) {
+            Debug.assert(!nodeIsMissing(currentSourceFile),

```

Comment:
`nodeIsMissing` does a lot of extra checks that aren't needed here.

```suggestion
            Debug.assertIsDefined(currentSourceFile,
```